src/axolotl/utils/models.py

```diff
@@ -234,6 +234,10 @@ def load_model(
                 base_model,
                 trust_remote_code=cfg.trust_remote_code or False,
             )
+            # Shouldn't be a problem most of the time. will obviously error if the model doesn't support this
+            # when training starts
+            if config.max_seq_len and cfg.sequence_len > config.max_seq_len:
```

Comment:
this will allow for specific cases to extend the context length (like MPT)

Reply:
I recommend adding a warning so people know this is happening

Reply:
thanks, I'll add it in the next round